### ISSUE TITLE
Zero inputs tried for bounded floats

### ISSUE THREAD

**Issue by tybug** [pschanely/hypothesis-crosshair#8]
This doesn't find an error:

```python
@settings(backend="crosshair", deadline=None, database=None)
@given(st.floats(1.0, 2.0)) # or other bounds
def f(x):
    assert False
f()
```

I see lots of `IgnoreAttempt` in debug logs. Probably crosshair is giving up generating inputs here when it shouldn't be?

**Comment by Zac-HD** [comment]
Ah, [leading asserts are treated as invariants](https://crosshair.readthedocs.io/en/latest/kinds_of_contracts.html#analysis-kind-asserts) - we should probably disable this for Hypothesis mode!

**Comment by pschanely** [comment]
OH - thanks! I think there is something else going on here. Looks like CrossHair never properly intercepted `math.isnan` to handle symbolics, and that call in the plugin is realizing the float prematurely. I'll fix this weekend!

**Comment by pschanely** [comment]
Should be fixed in [0.0.6](https://pypi.org/project/hypothesis-crosshair/0.0.6/). Please reopen if it isn't!
